src/lib.rs

```diff
@@ -290,5 +449,32 @@ mod tests {
         assert!(vhal
             .recv_cmd()
             .is_ok_and(|cmd| cmd.has_status() && cmd.status() == Status::RESULT_OK));
+        let recv = vhal.get_gear_selection();
```

Comment:
Better name? Like `gear` or `gear_selection`?
Or you intentionally wanted it to be a generic `recv`?